src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/Shared/TestTelemetryReporter.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Telemetry;
+using Microsoft.VisualStudio.Telemetry;
+
+namespace Microsoft.VisualStudio.Editor.Razor.Test.Shared;
+internal class TestTelemetryReporter() : TelemetryReporter(default)
```

Comment:
nit: please add a blank line between namespace and class.

Reply:
Add new file doesn't do that :( fixed

Reply:
Thanks! It helps me read. :smile: